framework/src/userobjects/Terminator.C

```diff
@@ -85,6 +85,24 @@ Terminator::Terminator(const InputParameters & parameters)
   _params.resize(_pp_num);
 }
 
+void
+Terminator::initialize()
```

Comment:
Could this be run on `initialSetup` instead of `initialize`?

Reply:
oh yeah